src/edg.rs

```diff
@@ -286,6 +387,7 @@ impl<B: Broker<V> + Debug, G: ExtendedDependencyGraph<V> + Send + Sync + Debug,
                 self.vertex_owner(vertex),
                 Message::REQUEST {
                     vertex: vertex.clone(),
+                    distance: self.distances.get(vertex).unwrap_or(&0).clone(),
```

Comment:
Fixes `clippy` warning
```suggestion
                    distance: self.distances.get(vertex).unwrap_or(&0),
```